packages/utils/src/address.ts

```diff
@@ -12,4 +12,12 @@ export const privateKeyToAddress = (privateKey: string) => {
   )
 }
 
+export const publicKeyToAddress = (publicKey: string) => {
+  return '0x' + pubToAddress(Buffer.from(publicKey.slice(2), 'hex')).toString('hex')
```

Comment:
Pretty sure web3 has a util for this already.

Reply:
Can you link it? I couldn't see it in particular? Also, I think we have generally tried to avoid depending on web3 beyond web3-utils in `utils`

Reply:
Yeah good point, nvm